image/directory/directory_dest.go

```diff
@@ -20,20 +20,73 @@ import (
 	"go.podman.io/storage/pkg/fileutils"
 )
 
-const version = "Directory Transport Version: 1.1\n"
+const (
```

Comment:
(Moving the `version` type, constants, and corresponding tests to `version.go` would not hurt.)

Reply:
done